packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -411,7 +327,7 @@ const DataGrid = injectIntl((props) => {
             || (targetElement.hasAttribute('contentEditable') && targetElement.getAttribute('contentEditable') !== false))) {
       return;
     }
-
+    const columns = tableRef.current.tHead.rows[0].cells;
```

Comment:
We should not need to access tHead.  Also, we already have a reference to our number of columns in state.  We don't need to access the DOM to get this information as we have a React component.

Reply:
Remove the tHead reference.